prov/efa/src/efa_cq.c

```diff
@@ -175,8 +179,13 @@ ssize_t efa_cq_readfrom(struct fid_cq *cq_fid, void *buf, size_t count,
 								  wc.ibv_wc.slid,
 								  wc.ibv_wc.src_qp);
 		}
-		cq->read_entry(&wc, i, buf);
-	}
+		cq->read_entry(&wc, i++, buf);
```

Comment:
instead of use `i`, let us use a more meaningful variable name such as `num_cqe`.

Reply:
Sure. Will address.